Add mission to take refugees to Humanika (#9159)
endless-sky/endless-sky: data/human/free worlds side plots.txt

		conversation
			`You receive a message from Freya: "Hello again, Captain! Thanks for your help escorting the materials to Solace. Ever since we developed the Dreadnoughts people have been talking about how badly we need a generator that can meet their energy needs, and that's what Delta V Corporation has been working on. So, take a look next time you're on Solace or one of our other major manufacturing centers. Sorry for the mystery surrounding the cargo, but we didn't exactly want to advertise that a huge shipment of fissionable material was headed for Free Worlds space."`
				decline

mission "FW Refugees to Humanika"
	name "Transport Refugee Family"
	description "Transport a family of <bunks> refugees and <cargo> to <destination>. Payment is <payment>."
	minor
	source
		attributes "dirt belt" rim south
		not government Quarg
		not near Tarazed 1
	destination Humanika
	to offer
		has "event: war begins"
		not "chosen sides"
		random < 15
	to fail
		has "event: joined the free worlds"
	passengers 3
	cargo "household goods" 2
	on offer
		conversation

> 			`A young family of <bunks> is walking around the spaceport talking to merchant captains. From the looks of it, they're not having much luck.`

Reviewer: Writing it out in text is better in this situation.
```suggestion
			`A young family of three is walking around the spaceport talking to merchant captains. From the looks of it, they're not having much luck.`
```

Author: Done. Just out of curiosity though, why is that better? (They seem about equally good to me, so happy to go with your suggestion regardless of the reasoning.)

			choice
				`	(Approach them.)`
				`	(Ignore them.)`
					defer

> 			`	The parents introduce themselves as Penny Little and Rosa Perkins, and their young child, Eduardo. "Any chance you're willing to take us and <tons> of our belongings to the Quarg planet <destination>? We're concerned that the war between the Republic and the Free Worlds is going to get dangerous, and we want to raise Eduardo somewhere safe. We can pay <payment>."`

Reviewer 2: ```suggestion
			`	The parents introduce themselves as Penny Little and Rosa Perkins. Their young child is named Eduardo. "Any chance you're willing to take us and <tons> of our belongings to the Quarg planet <destination>? We're concerned that the war between the Republic and the Free Worlds is going to get dangerous, and we want to raise Eduardo somewhere safe. We can pay <payment>."`
```

Author: Done.

> 			`	The parents introduce themselves as Penny Little and Rosa Perkins. Their young child is named Eduardo. "Any chance you're willing to take us and <tons> of our belongings to the Quarg planet <destination>? We're concerned that the war between the Republic and the Free Worlds is going to get dangerous, and we want to raise Eduardo somewhere safe. We can pay <payment>."`

Author: > You should clarify which one of them is speaking to the player.

Does it matter? It seemed irrelevant to me, so I didn't include it.

> `Quarg planet of <destination>`

Isn't either way correct?

> 			`	The parents introduce themselves as Penny Little and Rosa Perkins. Their young child is named Eduardo. "Any chance you're willing to take us and <tons> of our belongings to the Quarg planet <destination>? We're concerned that the war between the Republic and the Free Worlds is going to get dangerous, and we want to raise Eduardo somewhere safe. We can pay <payment>."`

Reviewer: There isn't anything we gain by not specifying the speaker.

"The Quarg planet of Humanika" is more aesthetically pleasing. Compare "the American city of Chicago" with "the American city Chicago".

> 			`	The parents introduce themselves as Penny Little and Rosa Perkins. Their young child is named Eduardo. "Any chance you're willing to take us and <tons> of our belongings to the Quarg planet <destination>? We're concerned that the war between the Republic and the Free Worlds is going to get dangerous, and we want to raise Eduardo somewhere safe. We can pay <payment>."`

Author: > There isn't anything we gain by not specifying the speaker.

Or by specifying the speaker either, so better to not give information that doesn't add anything, right? That way if there's a reason it should be one speaker or the other in the future, it can be specified then without changing anything that already happened.

> "The Quarg planet of Humanika" is more aesthetically pleasing. Compare "the American city of Chicago" with "the American city Chicago".

I don't think I would ever use "of" in "the American city of Chicago" either. (It's correct, just not something I'd say/write.) Is there a relevant style guide to resolve this?

			label "main choice"
			choice
				`	"Why were all those other captains refusing to take you?"`
					to display
						not "FW Refugees to Humanika: other captains"
					goto "other captains"

> 				`	"What do you mean, 'Quarg planet'?"`

Reviewer: ```suggestion
				`	"What do you mean, 'Quarg planet?'"`
```

Author: `'Quarg planet'` is not a question, so I think the question mark belongs outside of that quote. https://www.thesaurus.com/e/grammar/does-punctuation-go-inside-or-outside-quotation-marks/
> If a sentence is a question or an exclamation, but the quote is not, we typically place the punctuation mark outside of the quotation marks.

					to display
						not "FW Refugees to Humanika: quarg"
						not "First Contact: Quarg: offered"
					goto quarg
				`	"The Quarg let humans live on one of their planets?"`
					to display
						not "FW Refugees to Humanika: quarg"
						has "First Contact: Quarg: offered"
						not "visited planet: Humanika"
					goto quarg
				`	"Sure, I can take you."`
					accept
				`	"Sorry, I'm not headed that way."`
					decline
			label "other captains"
			action
				set "FW Refugees to Humanika: other captains"
			`	"I think many of them are a bit distrustful of the Quarg, and people are just generally more on edge these days with the tensions between the Free Worlds and the Republic, so they're less interested in flying somewhere unfamiliar. While I do understand those feelings, it seems to me that humans have done much worse to each other than the Quarg have ever done to us."`
				goto "main choice"
			label "quarg"
			action
				set "FW Refugees to Humanika: quarg"

> 			`	"The Quarg are an advanced and apparently peaceful alien species that live in a corner of human space, near Tarazed. I honestly don't know that much else about them, but I've heard they're happy to talk to anybody who visits them. You could try that if you want to know more."`

Reviewer 3: Perhaps a "she looks surprise at you not knowing who tf the big scary aliens down east are for some guy who lived in the rim for so long-"

Author: Good idea, how's this?

For explanation of why I changed your suggestion a bit: I had tried to word the question (`"What do you mean, 'Quarg planet'?"`) ambiguously so it's not clear if the player is asking who the Quarg are, or asking what's going on with them having a planet instead of a moon/station. So I toned down the surprise a bit, because not knowing about the planet seems much less surprising than not knowing about the quarg at all. Also, presumably she doesn't think they're "big scary" if she wants to go live on their planet.

				to display
					not "First Contact: Quarg: offered"
			`	"There's a planet they control that has too much gravity for them, but is just right for humans, so they let humans settle there under their protection. Or at least that's what I've heard."`
				goto "main choice"
	on visit
		dialog phrase "generic cargo and passenger on visit"
	on fail

> 		dialog "Penny Little and Rosa Perkins let you know that they wanted to get to <destination> to avoid the hostilities, so they'll find another captain who's still neutral."

Reviewer 3: ```suggestion
		dialog "Penny and Rosa let you know that they wanted to get to <destination> to avoid the hostilities, so they'll find another captain who's still neutral."
```

Reviewer 3: There's no need to name their full names every time. The first would've been enough.

Hell, there _may_ not even be a need to give their surnames, but that's a topic I'm not keen to go down on.

Author: Makes sense, done.

> 		dialog "Penny and Rosa let you know that they wanted to get to <destination> to avoid the hostilities, so they'll find another captain who's still neutral."

Author: Isn't hostilities more correct here? The reason they abandon the player isn't because of an official declaration of war, but because the player put them in the path of the hostilities of that war.

> 		dialog "Penny and Rosa let you know that they wanted to get to <destination> to avoid the hostilities, so they'll find another captain who's still neutral."

Reviewer 2: Yeah, good point. You can ignore this change.

> 		dialog "Penny and Rosa let you know that they wanted to get to <destination> to avoid the hostilities, so they'll find another captain who's still neutral."

Reviewer: This isn't necessary. It also doesn't make sense if the mission is failed by, say, having the ship they're travelling in get blown up.

Author: > This isn't necessary. 

Not necessary, but it seems pretty important to me. These are people who are presumably putting in a significant amount of effort to avoid the risks of the war, so there's no way they'd want to get caught up in it. And it seems like they should get pretty annoyed if a captain agreed to take them away from the war, but took them towards it instead.

> It also doesn't make sense if the mission is failed by, say, having the ship they're travelling in get blown up.

Good point. Is there an easy way to handle that differently? I don't see a way to make dialogs or conversations conditional. (Conversations can have parts that are conditional, but I don't see how to make the whole thing conditional.)

Reviewer: `on fail` dialogue is reserved for only a few select situations, those being when failing the mission has severe consequences (such as softlocking the campaign or angering an entire faction) or when failing the mission provides an alternate mission ending/route. Any other time, we leave it up to implication that the people involved with the mission find another way.

Author: Got it, thank you for the explanation! (I left the `to fail` in, just removed the `on fail`. Let me know if I misunderstood.)

	on complete
		payment
		payment 20000

> 		dialog "Penny Little and Rosa Perkins thank you for getting them safely to <destination>. You wish them and Eduardo the best of luck starting their new life away from a potential war in human space."

Reviewer 3: ```suggestion
		dialog "Penny and Rosa thank you for getting them safely to <destination>, handing you <payment> for your troubles. You wish them and Eduardo the best of luck starting their new life away from a potential war in human space."
```

Reviewer 3: Consider adding dialogue here on thanking the Player.

Author: Done.

> 		log "Minor People" "Penny Little, Rosa Perkins, and Eduardo" `A young family who wanted to escape a potential war in human space. You took them to the Quarg planet Humanika, where the Quarg let humans settle.`

Reviewer 3: Logs are written in the first person by the Player Character. Would need this rewritten in that POV.

Author: Done, thanks for letting me know.

> 		log "Minor People" "Penny Little, Rosa Perkins, and Eduardo" `A young family who wanted to escape a potential war in human space. You took them to the Quarg planet Humanika, where the Quarg let humans settle.`

Reviewer 2: I think this would fix the log.
```suggestion
		log "Minor People" "Penny Little, Rosa Perkins, and Eduardo" `A young family who wanted to escape a potential war in human space. Took them to the Quarg planet Humanika, where the Quarg let humans settle.`
```

Author: Done.
